docs/madr/002-mainnet-bootstrapping.md

```diff
@@ -0,0 +1,120 @@
+Deceision outcome
```

Comment:
Nit: Spelling

Reply:
Honestly I think you can delete this file from the PR but its existence will still be present in the history.

Reply:
removed in [b501657](https://github.com/ar-io/ar-io-network-process/pull/388/commits/b50165766471e76ce85d798e388178ccc591bb1d)